src/lib/tvgArray.h

```diff
@@ -73,9 +73,14 @@ struct Array
         return data + count;
     }
 
-    T* last() const
+    T& last() const
```

Comment:
if the method is const, return a const ref.

Reply:
@vtorri thanks, but please patches! :)